.github/workflows/azure-devops-pipelines-cg.yml

```diff
@@ -37,7 +37,7 @@ extends:
         - task: UseDotNet@2
           inputs:
             packageType: 'sdk'
-            version: '6.x'
+            version: '8.x'
```

Comment:
Additionally, there are few references of `win10-x64` in release pipeline and `install.ps1`. Let's verify if we should update those to `win-x64` since they have changed the `runtime identifier from .NET8`

Reply:
oh good catch, i thought i updated this but i did not.. done

Reply:
Replaced the occurrences of win10-x64 with win-x64 in release yaml file and bin\win\publish.cmd.  Tested with release pipeline too.
Will replace the occurrences of win10-x64 in install/install.ps1 after release is done